src/app/ipv/middleware.js

```diff
@@ -347,16 +347,7 @@ async function updateJourneyState(req, res, next) {
   }
 }
 
-async function handleJourneyPage(req, res, next) {
-  return handleJourneyPageInternal(req, res, next);
-}
-
-async function handleJourneyPageInternal(
-  req,
-  res,
-  next,
-  pageErrorState = undefined,
-) {
+async function handleJourneyPage(req, res, next, pageErrorState = undefined) {
```

Comment:
given it's a boolean maybe `pageHasErrorState` and default to false?

Reply:
It's not always a boolean :(

Reply:
Although, I guess in this function it is.
I was trying to keep the existing behaviour where if there isn't an error then pageErrorState just doesn't exist in the render options